src/cache/cache_las.c

```diff
@@ -35,6 +35,17 @@ __las_restore_isolation(WT_SESSION_IMPL *session, WT_TXN_ISOLATION saved_isolati
     session->txn.isolation = saved_isolation;
 }
 
+/*
+ * __las_store_time_pair --
+ *     Store the time pair to use for the lookaside inserts.
+ */
+static void
+__las_store_time_pair(WT_SESSION_IMPL *session, wt_timestamp_t timestamp, uint64_t txnid)
+{
+    S2C(session)->cache->org_timestamp_to_las = timestamp;
```

Comment:
The common abbreviation for `original` is `orig` - would you mind making that change?

Reply:
Fixed.